### PR TITLE
Rename ocis-phoenix to ocis-web

### PR DESCRIPTION
**Pull request by LukasHirt**

### CODE REVIEW

**Comment by LukasHirt** [comment]
There were some duplicate `WEB_WEB` vars. For now, they are shortened to `WEB`. I would propose though to change it to `WEB_UI_*`

File: docs/extensions/web/releasing.md [hunk 1]
```diff
@@ -2,37 +2,37 @@
...
+geekdocEditPath: edit/master/docs/extensions/web
 geekdocFilePath: releasing.md
 ---
 
 {{< toc >}}
 
 ## Releasing
 
-The next generation Web Frontend is shipped as an oCIS Extension. The `ocis-phoenix` extension is also embedded in the single binary and part of the `ocis server` command.
+The next generation Web Frontend is shipped as an oCIS Extension. The `ocis-web` extension is also embedded in the single binary and part of the `ocis server` command.
```
Comment (phil-davis) [inline docs/extensions/web/releasing.md:13]:
```suggestion
The next generation Web Frontend is shipped as an oCIS Extension. The `web` extension is also embedded in the single binary and part of the `ocis server` command.
```

Now I am not sure if the name here should be `ocis-web` or just `web`

**Review by phil-davis — COMMENTED** [review]
There are quite a few mentions of `ocis-web` - are those expected?

**Comment by LukasHirt** [comment]
> There are quite a few mentions of ocis-web - are those expected?

I left it in some places to make a difference between ownCloud Web and the ocis extension. If you'd see places where it still would make more sense to use ownCloud Web or only Web, I'd be happy to chage those 🙂

**Comment by phil-davis** [comment]
@LukasHirt I see that you found the similar changes to what I made in PR #1086 

I think you will also need to bump the commit id like https://github.com/owncloud/ocis/pull/1086/commits/70436c9401e1ff9bc59d088f45e10c7debe2c378 so that CI runs the newest `owncloud/web` test code that uses the `WEB_UI_CONFIG` environment variable.

A rebase to get the drone changes from #1087 would also be good - that makes sure that the correct test suites are actually running and passing.

**Comment by LukasHirt** [comment]
@phil-davis Thanks for letting me know! Completely forgot about bumping the id... I'll do so now together with the rebase :wink:

**Review by phil-davis — APPROVED** [review]
Looks good.

**Comment by phil-davis** [comment]
@LukasHirt IMO this can be merged. Then we can straight away work on whatever needs to change next in `owncloud/web` for it to run CI against "the new OCIS".

**Pull request merged**